src/day-1-toy/lists.py

```diff
@@ -11,19 +11,22 @@
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# [command here]
+x.append(4)
```

Comment:
This is correct, but the code is under the wrong problem statement. For this particular problem statement, you can use ```x.extend(y)``` to add all the elements in list y to list x in one go.